### PR TITLE
Jellis/efs docs

### PR DESCRIPTION
**Pull request by ellisjoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,48 @@ which stores the key for future access. The provided storage strategy
...
+```xml
+<configuration>
+    <property>
+        <name>fs.efile.impl</name> <!-- others: fs.es3a.impl or fs.ehdfs.impl -->
+        <value>com.palantir.hadoop.ConfigurableEncryptedFileSystem</value>
+    </property>
+
+    <property>
+        <name>fs.efs.key.public</name>
+        <value>REDACTED</value>
```
Comment (uschi2000) [inline README.md]: Should mention format. Maybe just generate a pair and leave a comment saying to generate them?

File: README.md [hunk 2]
```diff
@@ -20,8 +20,48 @@ which stores the key for future access. The provided storage strategy
...
+    <property>
+        <name>fs.efs.key.private</name>
+        <value>REDACTED</value>
+    </property>
+</configuration>
+```
+
+##### Commands
+``` bash
+./bin/hadoop dfs -put file.txt /tmp/file.txt
```
Comment (uschi2000) [inline README.md]: Why would this use efile and not file? Maybe better to use paths with schema.

File: README.md (hunk 1 above)
Comment (uschi2000) [reply README.md]: I see you mention the format below. Would still be good to give an example and show how to generate then in linux

**Review by uschi2000 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ellisjoe) [reply README.md]: whoops. fixed

File: README.md (hunk 1 above)
Comment (ellisjoe) [reply README.md]: done

**Comment by uschi2000** [comment]
+1

**Pull request merged**